portal/src/components/Header/components/FullScreenMenuItem.scss

```diff
@@ -23,21 +23,13 @@
         background-color: transparent;
         cursor: pointer;
 
-        @include medium-device {
+        @include from-medium-device {
             // static font-sizes for old browsers:
             font-size: rem(35px);
-            line-height: rem(40px);
+            line-height: rem(50px);
             // dynamic font sizes
-            font-size: relativeSize(35, 768, 1024);
-            line-height: relativeSize(50, 768, 1024);
```

Comment:
Disse skal ikke settes ut fra oppløsningen på breakpoint, men ut fra oppløsningen på skissen (1440 x 900) for å få tiltenkt størrelse 🙂